pglite/cibuild/pgbuild.sh

```diff
@@ -93,35 +93,32 @@ CC_PGLITE=$CC_PGLITE
     if $WASI
     then
         export EXT=wasi
-        cat > ${PGROOT}/config.site <<END
-ac_cv_exeext=.wasi
-END
+        ACCVEXEEXT==.wasi
+
         if $GETZIC
         then
             cat > bin/zic <<END
 #!/bin/bash
 #. /opt/python-wasm-sdk/wasm32-wasi-shell.sh
-TZ=UTC PGTZ=UTC $(command -v wasi-run) $(pwd)/src/timezone/zic.wasi \$@
+TZ=UTC PGTZ=UTC $(which wasi-run) $(pwd)/src/timezone/zic.wasi \$@
 END
         fi
 
     else
         export EXT=wasm
-        cat > ${PGROOT}/config.site <<END
-ac_cv_exeext=.cjs
-END
+        ACCVEXEEXT=.cjs
 
         if $GETZIC
         then
             cat > bin/zic <<END
 #!/bin/bash
 #. /opt/python-wasm-sdk/wasm32-bi-emscripten-shell.sh
-TZ=UTC PGTZ=UTC $(command -v node) $(pwd)/src/timezone/zic.cjs \$@
+TZ=UTC PGTZ=UTC $(which node) $(pwd)/src/timezone/zic.cjs \$@
 END
         fi
     fi
 
-    if EM_PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig CONFIG_SITE=${PGROOT}/config.site emconfigure $CNF --with-template=$BUILD
+    if EM_PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig ac_cv_exeext=$ACCVEXEEXT emconfigure $CNF --with-template=$BUILD
```

Comment:
please do not scatter autoconf modifiers, keep them in one file passed with CONFIG_SITE. this may be need to get homogenous build of third party libraries